packages/pytest-simcore/src/pytest_simcore/docker_swarm.py

```diff
@@ -205,41 +214,44 @@ def docker_stack(
 
     # make down
     # NOTE: remove them in reverse order since stacks share common networks
-    WAIT_BEFORE_RETRY_SECS = 1
 
     HEADER = "{:-^20}"
     stacks.reverse()
     for _, stack, _ in stacks:
 
         try:
-            subprocess.run(f"docker stack remove {stack}", shell=True, check=True)
+            subprocess.run(
+                f"docker stack remove {stack}",
+                shell=True,
+                check=True,
+                capture_output=True,
+            )

```

Comment:
I would always capture the output from here, and always print it. Even if no error occurs.